pphtml.py

```diff
@@ -357,17 +357,18 @@ def imageSummary(self):
 
     def coverImage(self):
         """
-        cover image width must be >=650 px and height must be >= 1000 px
+        cover image width should be >=1600 px and height should be >= 2560 px
+        https://www.pgdp.net/wiki/DP_Official_Documentation:PP_and_PPV/Post-Processing_FAQ#Cover_image
         """
         r = ["[pass] cover image dimensions check"]
         for t in self.filedata:
             u = t.split("|")
-            if u[0] == "cover.jpg":
+            if u[0] == "cover.jpg" or u[0] == "cover.png":
```

Comment:
Suggestion: this would be more pythonic (here and below):
```suggestion
            if u[0] in ("cover.jpg", "cover.png"):
```